aptos-move/e2e-tests/goldens/language_e2e_testsuite__tests__module_publishing__bad_module_address.exp

```diff
@@ -0,0 +1,45 @@
+Ok(
```

Comment:
Hmm.. why these new files?
Did you accidentallly check these files back in after they were deleted?